docs/sql-programming-guide.md

```diff
@@ -1740,17 +1759,14 @@ results = spark.sql("FROM src SELECT key, value").collect()
 
 <div data-lang="r"  markdown="1">
 
-When working with Hive one must construct a `HiveContext`, which inherits from `SQLContext`, and
+When working with Hive one must construct a `HiveContext`, which inherits from `SparkSession`, and
```

Comment:
ditto here, `spark.session(enableHiveSupport = TRUE)` instead.
HiveContext has been deprecated in SparkR.